modmesh/app/cavity2d.py

```diff
@@ -0,0 +1,383 @@
+__package__ = "modmesh.app"
+
+import sys
+import numpy as np
+
+import matplotlib.pyplot as plt
+from matplotlib.backends.qt_compat import QtWidgets
+from matplotlib.backends.backend_qtagg import (
+    FigureCanvas, NavigationToolbar2QT as NavigationToolbar)
+from matplotlib.figure import Figure
+from .. import view
+
+
+def load_app():
+    view.mgr.pycon.writeToHistory("""
+# Use the functions for more examples:
+ctrl.start()  # Start computing. Multiple blank windows will appear, and the
+results will be displayed once the execution is terminated.
+<ctrl> + C # Terminate in terminal
+
+
+ctrl = mm.app.cavity2d.run(interval=10, max_steps=50, profiling=True)
+""")
+    cmd = "ctrl = mm.app.cavity2d.run(interval=10, max_steps=50)"
+    view.mgr.pycon.command = cmd
+
+
+###############################################################################
+N = 81
```

Comment:
Please avoid using so many global variables.  Instead, create a class to hold all the values and pass it to the functions or classes that need them.